### PR TITLE
添加更多翻译（新页面：Github Copilot页）

### PR DESCRIPTION
**Pull request by TC999**
添加对Github Copilot页面的部分翻译（不完全参照机翻，根据实际语境进行修改）
简化一些词条
注：未匹配Copilot页正则

### CODE REVIEW

File: locals.js [hunk 1]
```diff
@@ -425,62 +423,31 @@ I18N.zh["pubilc"] = { // 公共区域翻译
...
+        "Reference an issue or pull request": "引用议题或拉取请求",
+        "Add saved reply": "添加快捷回复",
+            "Select a reply": "选择回复",
+            "Filter replies…": "筛选回复",
+            "Default replies": "默认快捷回复",
+            "Duplicate issue": "重复议题",
+            // "Duplicate of #": "重复 #",
+            "Duplicate pull request": "重复拉取请求",
+            // "Duplicate of #": "重复 #",
+            "Create a new saved reply…": "创建新快捷回复…",
```
Comment (maboloshi) [inline locals.js:450]: 你这边显示的还是老版编辑器?

**Comment by wuyuncheng-26** [comment]
@TC999 请查阅一下

**Comment by TC999** [comment]
已修复，一会重投

**Comment by wuyuncheng-26** [comment]
？

**Comment by wuyuncheng-26** [comment]
直接在 fork 的仓库里提交就行了，这里会显示的

**Comment by maboloshi** [comment]
请重新复刻在提交, 代码乱掉了

**Comment by maboloshi** [comment]
Github Copilot页面 链接发我一下 @TC999

**Comment by TC999** [comment]
I18N.zh["github-copilot/signup"] = { //COPILOT页面
    "static":{ // 静态翻译
       "Your AI pair programmer":"您的AI编程助手",
       "GitHub Copilot uses the GPT-3.5 Turbo model to offer coding suggestions.":"GitHub Copilot使用GPT-3.5 Turbo模型提供编程建议。",
       "Get code suggestions in more than a dozen coding languages including Python, JavaScript, TypeScript, Go, and Ruby":"支持Python、JavaScript、TypeScript、Go和Ruby等十几种编程语言",
       "Plugs into IDEs including VS Code, Visual Studio, Neovim, and JetBrains":"可在VS Code,Visual Studio，Neovim和JetBrains等IDE中调用",
       "Sign up for an annual subscription and get two months free!":"注册年费会员，即享受两个月免费试用！",
       "Cancel anytime before the trial ends to avoid getting billed":"试用结束前可随时取消",
       "Pay frequency after trial":"试用期结束后",
       "Monthly plan":"月度套餐",
       "Yearly plan":"年度套餐",
       "Save":"节省",
       "Get access to GitHub Copilot":"获取GitHub Copilot访问权限",
       "Next: Confirm your payment details":"下一步：确认您的付款详细信息",
       "GitHub Copilot is free for maintainers of popular open source software on GitHub and verified students. Check out the":"GitHub Copilot 对 GitHub 上流行开源软件的维护者和经过验证的学生免费。查看",
       "free use FAQ":"免费使用常见问题解答",
       "for more details.":"以了解更多详细信息。",
    },
};

**Comment by wuyuncheng-26** [comment]
这个吧 <https://github.com/github-copilot/signup>

**Comment by TC999** [comment]
> 这个吧 https://github.com/github-copilot/signup

是的

**Pull request closed without merging**